app/containers/Questions/constants.js

```diff
@@ -12,4 +12,6 @@ export const GET_UNIQ_QUESTIONS = 'app/Questions/GET_UNIQ_QUESTIONS';
 export const SET_TYPE_FILTER = 'app/Questions/SET_TYPE_FILTER';
 export const SET_CREATED_FILTER = 'app/Questions/SET_CREATED_FILTER';
 
-export const TOP_COMMUNITY_DISPLAY_MIN_RATING = !process.env.IS_TEST_ENV ? 10 : 3;
+export const TOP_COMMUNITY_DISPLAY_MIN_RATING = !process.env.IS_TEST_ENV
```

Comment:
It is better to add that variable into .env file instead of adding conditional logic here